### PR TITLE
Feature/sc 52

### PR DESCRIPTION
**Pull request by DanCh11**
Changed commented issues

### CODE REVIEW

File: notebooks/pwdb/algorithms/KNN/law2vec/target_groups_classes.ipynb [hunk 1]
```diff
@@ -2,10 +2,7 @@
  "cells": [
```
Comment (costezki) [inline notebooks/pwdb/algorithms/KNN/law2vec/target_groups_classes.ipynb:2]: <p>looks elegant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ml-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (costezki) [inline notebooks/pwdb/algorithms/KNN/law2vec/target_groups_classes.ipynb:2]: <p>looks reliable </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ml-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (costezki) [inline notebooks/pwdb/algorithms/KNN/law2vec/target_groups_classes.ipynb:2]: <p>the function looks more readable </p><p>the parameters look confusing</p><p> </p><p>Shouldn't the columns be <code>"Metrics labels"</code> and <code>"Metrics values"</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ml-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (costezki) [inline notebooks/pwdb/algorithms/KNN/law2vec/target_groups_classes.ipynb:2]: <p>looks reliable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ml-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

File: notebooks/pwdb/algorithms/KNN/law2vec/target_groups_subclasses.ipynb [hunk 2]
```diff
@@ -41,30 +41,43 @@
     "\n",
```
Comment (costezki) [inline notebooks/pwdb/algorithms/KNN/law2vec/target_groups_subclasses.ipynb:41]: <p>I guess <code>TargetGroups</code>  is Metric Values, isn't it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ml-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: notebooks/pwdb/algorithms/SVM/category.ipynb [hunk 3]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (costezki) [inline notebooks/pwdb/algorithms/SVM/category.ipynb:2]: <p>consistent usage of parameters looks reliable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ml-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

File: notebooks/pwdb/algorithms/SVM/law2vec/category.ipynb [hunk 4]
```diff
@@ -30,32 +30,44 @@
     "\n",
```
Comment (costezki) [inline notebooks/pwdb/algorithms/SVM/law2vec/category.ipynb:30]: <p>comments can be useful</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ml-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: sc_wrangling/mean_vectorizer.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from gensim.models import Word2Vec
+
+import pathlib
+FOLDER = pathlib.Path("/home/jovyan/work/upload/")
+LAW2VEC_PATH = '/home/jovyan/work/upload/data/law2vec/Law2Vec.200d.txt'
+
+# Insert our data and set minimal word count to 10, and size of each word to 300 vectors
+model = Word2Vec.load('/home/jovyan/work/Dan/data/pwdb/word2vec/df.model')
```
Comment (costezki) [inline sc_wrangling/mean_vectorizer.py:9]: - not a good idea to load the model into the module-level variable. 
- define PATH constants instead
- watch out, the folder DAN will soon be deleted.

File: sc_wrangling/mean_vectorizer.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+w2v_dict = {w: vec for w, vec in zip(model.wv.index2word, model.wv.syn0)}
```
Comment (costezki) [inline sc_wrangling/mean_vectorizer.py:10]: this should not be done at the module level

File: sc_wrangling/mean_vectorizer.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class MeanEmbeddingVectorizerWord2Vec(object):
```
Comment (costezki) [inline sc_wrangling/mean_vectorizer.py:13]: This class seem to be duplicated with the obe below. Why? 
What is the main difference between them? 
Would we instead define a single class? 
What is the function of `self.word2vec`?

**Review by costezki — APPROVED** [review]
the code looks much better now.
What is the difference between `MeanEmbeddingVectorizerWord2Vec` and `MeanEmbeddingVectorizerWord2Vec` ?

File: sc_wrangling/mean_vectorizer.py (hunk 5 above)
Comment (DanCh11) [reply sc_wrangling/mean_vectorizer.py:9]: I have an issue with calling the path from a constant variable

File: sc_wrangling/mean_vectorizer.py (hunk 7 above)
Comment (DanCh11) [reply sc_wrangling/mean_vectorizer.py:13]: i tried, but both classes use different word embeddings

**Pull request merged**